src/test/java/com/razorpay/OrderClientTest.java

```diff
@@ -0,0 +1,121 @@
+        }
+    }
+
+    /**
+     * Retrieve the order details using order id.
+     * @throws RazorpayException
+     */
+    @Test
+    public void fetch() throws RazorpayException{
+        String mockedResponseJson = "{\"id\":"+ORDER_ID+",\"entity\":\"order\",\"amount\":2200,\"amount_paid\":0,\"amount_due\":2200,\"currency\":\"INR\",\"receipt\":\"Receipt#211\",\"status\":\"attempted\",\"attempts\":1,\"notes\":[],\"created_at\":1572505143}";
+        try {
+            mockResponseFromExternalClient(mockedResponseJson);
+            mockResponseHTTPCodeFromExternalClient(200);
+            Order fetch = orderClient.fetch(ORDER_ID);
+            assertNotNull(fetch);
+            assertEquals(true,fetch.has("id"));
+            assertTrue(fetch.has("entity"));
+            assertTrue(fetch.has("amount"));
+            assertTrue(fetch.has("amount_paid"));
+        } catch (IOException e) {
+            assertTrue(false);
+        }
+    }
+
+    /**
+     * Retrieve the payment details of respective customer using payment id.
+     * @throws RazorpayException
+     */
+    @Test
+    public void fetchPayments() throws RazorpayException{
+        String mockedResponseJson = "{\"entity\":\"collection\",\"count\":1,\"items\":[{\"id\":\"pay_DaaSOvhgcOfzgR\",\"entity\":\"payment\",\"amount\":2200,\"currency\":\"INR\",\"status\":\"captured\",\"order_id\":\"order_DaaS6LOUAASb7Y\",\"invoice_id\":null,\"international\":false,\"method\":\"card\",\"amount_refunded\":0,\"refund_status\":null,\"captured\":true,\"description\":\"Beansineveryimaginableflavour\",\"card_id\":\"card_DZon6fd8J3IcA2\",\"bank\":null,\"wallet\":null,\"vpa\":null,\"email\":\"gaurav.kumar@example.com\",\"contact\":\"+919999999988\",\"notes\":[],\"fee\":44,\"tax\":0,\"error_code\":null,\"error_description\":null,\"created_at\":1572505160}]}";
+        try {
+            mockResponseFromExternalClient(mockedResponseJson);
+            mockResponseHTTPCodeFromExternalClient(200);
+            List<Payment> fetch = orderClient.fetchPayments(ORDER_ID);
+            assertNotNull(fetch);
+            assertTrue(fetch.get(0).has("id"));
+            assertTrue(fetch.get(0).has("amount"));
+            assertTrue(fetch.get(0).has("currency"));
+        } catch (IOException e) {
+            assertTrue(false);
+        }
+    }
+
+    /**
+     * Update an order of respective customer using order id with object of that properties
+     * @throws RazorpayException
+     */
+    @Test
+    public void edit() throws RazorpayException {
+        JSONObject request = new JSONObject("{\"notes\":{\"key1\":\"value3\",\"key2\":\"value2\"}}");
+        String mockedResponseJson = "{\"id\":"+ORDER_ID+",\"entity\":\"order\",\"amount\":2200,\"amount_paid\":0,\"amount_due\":2200,\"currency\":\"INR\",\"receipt\":\"Receipt#211\",\"offer_id\":null,\"status\":\"attempted\",\"attempts\":1,\"notes\":{\"notes_key_1\":\"Tea,EarlGrey,Hot\",\"notes_key_2\":\"Tea,EarlGrey…decaf.\"},\"created_at\":1572505143}";
```

Comment:
some jsons string are used in multiple methods, can create private method

Reply:
ok i will do that .